gc/gc-tool/src/main/java/org/projectnessie/gc/tool/cli/options/IcebergOptions.java

```diff
@@ -24,14 +24,56 @@ public class IcebergOptions {
   @CommandLine.Option(
       names = {"-I", "--iceberg"},
       split = ",",
-      description = "Iceberg properties used to configure the FileIO.")
+      description = {
+        "Iceberg properties used to configure the FileIO.",
+        "The following properties are almost always required.",
+        "",
+        "For S3:",
+        "- s3.access-key-id",
+        "- s3.secret-access-key",
+        "- s3.endpoint, if you use an S3 compatible object store like MinIO",
+        "",
+        "For GCS:",
+        "- io-impl=org.apache.iceberg.gcp.gcs.GCSFileIO",
+        "- gcs.project-id",
+        "- gcs.oauth2.token",
+        "",
+        "For ADLS:",
+        "- io-impl=org.apache.iceberg.azure.adlsv2.ADLSFileIO",
+        "- adls.auth.shared-key.account.name",
+        "- adls.auth.shared-key.account.key",
+      })
   Map<String, String> icebergProperties = new HashMap<>();
 
   @CommandLine.Option(
       names = {"-H", "--hadoop"},
       split = ",",
-      description =
-          "Hadoop configuration option, required when using an Iceberg FileIO that is not S3.")
+      description = {
+        "Hadoop configuration option, required when using an Iceberg FileIO that is not S3.",
```

Comment:
maybe `S3` -> `S3FileIO`?.. otherwise it looks confusing wrt to the following S3 examples.